### DISCUSSION TITLE
DeepSeek

### DISCUSSION THREAD

**Discussion by den47999** [lobehub/lobehub#1800] [Ideas]
Добрый день:) добавьте https://platform.deepseek.com/docs  новую модель DeepSeek:  deepseek-chat, deepseek-coder

**Comment by MapleEve** [comment]
https://github.com/lobehub/lobe-chat/discussions/1284

You can post it here, anyway DeepSeek uses OpenAI api, so you can currently use any other local API gateway like LiteLLM, One-api, New-api to migrate them.

**Reply by JOduMonT** [reply]
> #1284
> 
> You can post it here, anyway DeepSeek uses OpenAI api, so you can currently use any other local API gateway like LiteLLM, One-api, New-api to migrate them.

@MapleEve would you mind to explain how to connect lobe to **LiteLLM** ?
because I connected my **[Claude Subscription](https://docs.litellm.ai/docs/tutorials/claude_code_max_subscription)** via **LiteLLM** and would like to **Lobe**

**Reply by MapleEve** [reply]
Use openai compatible
